doc/whats_new.rst

```diff
@@ -16,6 +16,13 @@ CLI
 API
 ---
 
+- Add :class:`~benchopt.stopping_criterion.SingleRunCriterion` to run a solver
+  only once. This can be used for benchmarking methods where we are interested
+  in objective value at convergence. By `Thomas Moreau`_ (:gh:`511`)
+- Add :func:`~benchopt.BaseSolver.warmup_solver` hook for to control solvers
```

Comment:
```suggestion
- Add :func:`~benchopt.BaseSolver.warmup_solver` hook to control solvers
```

Reply:
This should actually be removed as we do not have the `warmup_solver` in the chosen solution.